validator/delphi_validator/validate.py

```diff
@@ -829,6 +978,29 @@ def validate(self, export_dir):
                         "reference data is empty; comparative checks could not be performed"))
                     continue
 
+
+                
+                
+                # Source frame with the day before and after
+                next_cutoff_date = checking_date + recent_lookbehind
+                source_prev_next_df = geo_sig_df.query(
+                    'time_value <= @next_cutoff_date & time_value >= @recent_cutoff_date')
+
+
+                earliest_available_date = source_prev_next_df["time_value"].min()
```

Comment:
I think they need to be completely continuous unfortunately, otherwise you will not get good results with the rolling frame. Maybe this is something we can discuss.